rfcs/0100-sign-commits.md

```diff
@@ -0,0 +1,326 @@
+---
```

Comment:
IMHO we should drop flake specifics until that has been standardized.

Reply:
Without flakes this would not work at all. I don't think there's anything that will change wrt. to the parts this RFC relies on.

Reply:
> Without flakes this would not work at all.

Guix proves you wrong here.

Would you mind explaining why do you consider flakes as a hard dependency here?

Reply:
Because currently channels use tarballs. You'd have to replace channels with something like flake inputs.

Reply:
I don't see what's preventing us to download the git history instead of a release talball. The channels are aware of the git commit hash they've been cut from.

Reply:
There is nothing preventing us from doing that, but that's a feature that doesn't exist in Nix right now, since you'd replace our current channels with a new Git-based channel system. If you have some fleshed out idea I'll add it to the alternatives part.